13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -207,7 +207,7 @@
     "#lumi = 4.7 # fb-1 # data_D only\n",
     "lumi = 10 # fb-1 # data_A,data_B,data_C,data_D\n",
     "\n",
-    "fraction = 1.0 # reduce this is if you want the code to run quicker\n",
+    "fraction = 0.1 # reduce this is if you want the code to run quicker\n",
```

Comment:
We might want to update the comment next to the fraction in "increase this to run on more events" or something similar, since before `fration` was set at 1.0.

Reply:
HZZAnalysis was just my test notebook, definitely needs to be reverted.